docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -57,78 +67,78 @@ To determine whether Office 2016 or Office 2019 is retail or volume-licensed, us
-| Office 2016,<br>Office 2019 | Retail | Yes<sup>1</sup> | Microsoft Edge<sup>2</sup> with WebView2 (Chromium-based) |
-| Office 2016,<br>Office 2019 | Retail | No | Microsoft Edge Legacy (EdgeHTML)<sup>2, 3</sup></br>If Edge isn't installed, Internet Explorer 11 is used. |
-| Office 2019 | Volume-licensed | Doesn't matter | Internet Explorer 11 |
-| Office 2016 | Volume-licensed | Doesn't matter | Internet Explorer 11 |
-| Office 2013 | Doesn't matter | Doesn't matter | Internet Explorer 11 |
+| Office 2021 | Doesn't matter | Yes<sup>1</sup> | WebView2 (Microsoft Edge<sup>2</sup> Chromium-based) |
+| Office 2016,<br>Office 2019 | Retail | Yes<sup>1</sup> | WebView2 (Microsoft Edge<sup>2</sup> Chromium-based) |
+| Office 2016,<br>Office 2019 | Retail | No | EdgeHTML (Microsoft Edge Legacy)<sup>2, 3</sup></br>If Edge isn't installed, Trident+ (Internet Explorer 11) is used. |
+| Office 2019 | Volume-licensed | Doesn't matter | Trident+ (Internet Explorer 11) |
+| Office 2016 | Volume-licensed | Doesn't matter | Trident+ (Internet Explorer 11) |
+| Office 2013 | Doesn't matter | Doesn't matter | Trident+ (Internet Explorer 11) |
 
 <sup>1</sup> On Windows versions prior to Windows 11, the WebView2 control must be installed so that Office can embed it. It's installed with perpetual Office 2021 or later; but it isn't automatically installed with Microsoft Edge. If you have an earlier version of perpetual Office, use the instructions for installing the control at [Microsoft Edge WebView2 / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/).
 
-<sup>2</sup> When you use Microsoft Edge, the Windows Narrator (sometimes called a "screen reader") reads the `<title>` tag in the page that opens in the task pane. In Internet Explorer 11, the Narrator reads the title bar of the task pane, which comes from the **\<DisplayName\>** value in the add-in's manifest.
+<sup>2</sup> When you use either EdgeHTML or WebView2, the Windows Narrator (sometimes called a "screen reader") reads the `<title>` tag in the page that opens in the task pane. In Trident+, the Narrator reads the title bar of the task pane, which comes from the **\<DisplayName\>** value in the add-in's manifest.
 
-<sup>3</sup> If your add-in includes the **\<Runtimes\>** element in the manifest, then it will not use Microsoft Edge Legacy (EdgeHTML). If the conditions for using Microsoft Edge with WebView2 (Chromium-based) are met, then the add-in uses that browser. Otherwise, it uses Internet Explorer 11. For more information, see [Runtimes](/javascript/api/manifest/runtimes).
+<sup>3</sup> If your add-in includes the **\<Runtimes\>** element in the manifest, then it will not use EdgeHTML. If the conditions for using WebView2 are met, then the add-in uses that browser. Otherwise, it uses Trident+. For more information, see [Runtimes](/javascript/api/manifest/runtimes).
```

Comment:
```suggestion
<sup>3</sup> If your add-in includes the **\<Runtimes\>** element in the manifest, then it won't use EdgeHTML. If the conditions for using WebView2 are met, then the add-in uses that browser. Otherwise, it uses Trident+. For more information, see [Runtimes](/javascript/api/manifest/runtimes).
```